oauth2client/contrib/appengine.py

```diff
@@ -49,9 +49,19 @@
 # TODO(dhermes): Resolve import issue.
```

Comment:
Is this `TODO` being addressed by this very pull request? Or does this `TODO` refer to some other work item?